packages/react/src/primitives/TextArea/useAutoresizeTextarea.ts

```diff
@@ -7,7 +7,7 @@ export const useAutoresizeTextArea = (
 ): void => {
   useEffect(() => {
     const resizeTextArea = () => {
-      if (textAreaRef && value) {
+      if (textAreaRef) {
```

Comment:
The bug was we were using value in the conditional here, value could be falsey and we still want to re-trigger the effect. So value is in the dependency array, but removing it from here  to fix the bug